gloo/test/allgather_test.cc

```diff
@@ -28,9 +28,9 @@ class AllgatherTest : public BaseTest,
 
 TEST_P(AllgatherTest, VarNumPointer) {
   const auto transport = std::get<0>(GetParam());
-  const auto contextSize = std::get<0>(GetParam());
```

Comment:
was this a bug in existing Gloo code? Do you why this test didn't fail in CI?

Reply:
Yes it was and since `std::get<0>(GetParam());` returned `0`, the `contextSize ` was `0` and the tests just didn't do anything